lib/Parser/rterrors.h

```diff
@@ -374,3 +374,4 @@ RT_ERROR_MSG(WASMERR_InvalidGlobalRef, 5673, "", "Global initialization does not
 RT_ERROR_MSG(WASMERR_NeedMemoryObject, 5674, "%s is not a WebAssembly.Memory", "WebAssembly.Memory object expected", kjstTypeError, 0)
 RT_ERROR_MSG(WASMERR_InvalidTypeConversion, 5675, "Invalid WebAssembly type conversion %s to %s", "Invalid WebAssembly type conversion", kjstTypeError, 0)
 RT_ERROR_MSG(WASMERR_DivideByZero, 5676,    "",  "Division by zero", kjstError, 0)
+RT_ERROR_MSG(JSERR_CantRedefineProp, 5677, "Cannot redefine property '%s'", "Cannot redefine property", kjstTypeError, 0)
```

Comment:
That was my first choice as well, but I am trying to match the wording with other browser. Also SetItem can fail for other reasons, so it is better to be not-specific to configurable only.

---
In reply to: [87445471](https://github.com/Microsoft/ChakraCore/pull/1969#discussion_r87445471) [](ancestors = 87445471)